common/setups/rasr/pytorch_onnx_hybrid_system.py

```diff
@@ -0,0 +1,157 @@
+import copy
+import itertools
+from typing import Dict, List, Optional, Tuple, Union
+
+# -------------------- Sisyphus --------------------
+
+from sisyphus import tk
+
+# -------------------- Recipes --------------------
+
+import i6_core.rasr as rasr
+import i6_core.returnn as returnn
+
+from i6_core.returnn.flow import (
+    make_precomputed_hybrid_tf_feature_flow,
+    add_tf_flow_to_base_flow,
+)
+
+
+# -------------------- Init --------------------
+
+Path = tk.setup_path(__package__)
+
+from i6_core.mm import CreateDummyMixturesJob
+from i6_core.rasr import OnnxFeatureScorer
+
+from i6_experiments.common.setups.rasr.hybrid_system import HybridSystem
+
+from i6_experiments.users.hilmes.tools.onnx import ExportPyTorchModelToOnnxJob
```

Comment:
It is not allowed to reference user code in common, also this way of export is deprecated. This should better be discussed...

Reply:
agreed :D

> Since there are dependencies on user stuff and there might be a few open questions I ll add this for now as a draft PR so that people are aware that this exists..

:D

Reply:
reading would help :)